feat(SC-2603): Only add parameters with Service tech access level to …

ssirnio opened the pull request:
## Jira Story
[SC-2603](https://epcpower.atlassian.net/browse/SC-2603)

## About
- Select only `Service Tech` access level parameters when generating static modbus from CAN data. Needed since static modbus will only contain parameters with this access level.
- Also filter out empty CAN signals
- Context menu command to copy object name or UUID

## Associated Pull Requests

* [ ] _
* [ ] _

## Reproduction Steps

## Validation


[SC-2603]: https://epcpower.atlassian.net/browse/SC-2603?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

ssirnio committed:
feat(SC-2603): Only add parameters with Service tech access level to static modbus

ssirnio committed:
feat(SC-2603): Filter out empty CAN signals

ssirnio committed:
feat(SC-2603): context menu command to copy UUID or name

gordon-epc reviewed (approved):
Noted that it might make sense to not allow drag & drop from parameter tree to static modbus tree anymore, since static modbus tree is a subset of the CAN tree.

ssirnio committed:
feat(SC-2603): Forbid drag&drop from parameter tree to static modbus root

- This is to discourage creation of static modbus entries without valid
  CAN signals
- Existing entries can still be overwritten by drag&drop from parameters
  for convenience
